eng/illink.targets

```diff
@@ -1,5 +1,17 @@
 <Project>
 
+  <PropertyGroup>
+    <_ILLinkTasksSourceDir>$(ToolsProjectRoot)illink/src/ILLink.Tasks/</_ILLinkTasksSourceDir>
+    <ILLinkAnalyzersPropsPath>$(ToolsProjectRoot)illink/src/ILLink.RoslynAnalyzer/build/Microsoft.NET.ILLink.Analyzers.props</ILLinkAnalyzersPropsPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(_ILLinkTasksSourceDir)ILLink.Tasks.csproj"
+                      ReferenceOutputAssembly="false" />
```

Comment:
You need to add `PrivateAssets="all"` as otherwise packages would have a package dependency to the illink package.

Reply:
Also, make sure that the project is built in the same configuration as when doing a root build. By default, the referencing project's configuration flows to the ProjectReference project. We probably don't want that here.